### ISSUE TITLE
Cannot disable InputItem 

### ISSUE THREAD

**Issue by Lee931018** [ant-design/ant-design-mobile#2187]
### Version
2.0.3

### Environment
react native 0.50.3        android 7.1.1 Pixel emulator 

### Reproduction link
[https://github.com/Lee931018/AntdTest.git](https://github.com/Lee931018/AntdTest.git)

### Steps to reproduce
`
<List>
   <InputItem  disable/>
<List>
`
the InputItem is still can edit and nothing changed with no disable property used.

### What is expected?
The InputItem is disabled when use disable property.

### What is actually happening?
The InputItem is not disable and can still can edit when use disable property.

**Comment by bang88** [comment]
@Lee931018 If you want `disable` the Input you should use `editable={false}`  prop for React Native

**Comment by wjielim** [comment]
@bang88 using `editable={false}` prop does stop the user from changing the value, however it is completely different from what `disabled` should do i.e. `disabled` should changes the appearance of the input item to grey as shown in the demo to give the user a feel that the input item is grayed out and not editable.

**Comment by bang88** [comment]
@wjielim I agree. but `disabled` is not implemented in `react-native` side now.

**Comment by wjielim** [comment]
@bang88 thanks for the prompt feedback. I'm aware of this because it seems like `disabled` is not implemented in `TextareaItem` as well in `react-native`. I'm glad that this caught your attention and looking forward for this feature!
